userspace/engine/falco_utils.h

```diff
@@ -39,6 +43,25 @@ namespace falco
 namespace utils
 {
 
+// TODO interim helper methods -> shall be integrated into sinsp APIs
+std::unordered_set<uint32_t> get_syscalls_ppm_sc_codes(const std::unordered_set<std::string>& syscalls_names);
+std::unordered_set<std::string> erase_io_syscalls_ppm_sc_codes(std::unordered_set<uint32_t>& ppm_sc_of_interest, std::unordered_set<uint32_t>& io_ppm_sc_set);
+std::unordered_set<uint32_t> enforce_events(std::unordered_set<uint32_t> ppm_event_info_of_interest = {});
+std::unordered_set<uint32_t> enforce_io_ppm_sc_set(std::unordered_set<uint32_t> ppm_sc_set = {}); // needs EC_IO_OTHER removed in sinsp API
+// end interim helper methods
```

Comment:
https://github.com/falcosecurity/libs/pull/867, `enforce_io_ppm_sc_set` also has a bug with current libs, but I fixed it already in libs, and in addition now adjusting re EC_IO_OTHER, hence why it's currently re-defined.